File: Assignment1/smallprogram.c
#include <stdio.h>
#define MAX 40 // maximum name length

//function prototype
void display (char n[], int times);

//main function (does not need prototype) int 
main (void)
{	char name[MAX]; // user's name
int loop=0; // number of loops
	printf ("What is your name? ");
	gets(name); // get user's name 
	printf ("How many times shall I print it? ");
	scanf ("%d",&loop); // read no. of times
		display (name, loop);
		return (0); } // end of

////function display (n)ame so many (t)imes 
void display (char n[], int t)
	{	int i=0; for 
	(i=0; i<t; i++) 
	printf ("%s ",n);
		printf ("\n"); }// end of display()

File: Assignment1/smallprogram.java
import java.*; 
import java.io.*;

public class smallprogram
{
public static void main(String[] args)
{
	String opinion = " likes Linux! "; 
char first=' ', last=' '; // user's initials 
int loop = 20; // number of times to repeat
	try
{
	System.out.print("Your first initial? ");
first = (char)System.in.read();
	System.in.read();
	System.out.print("Your last initial? ");
last = (char)System.in.read();
	System.in.read();
}
catch (IOException e)
{
}

System.out.println("Repeating "+loop+" times.");
for (int i=0; i<loop; i++) 
	System.out.print(first+"."+last+"."+opinion);
System.out.println();
     } // end of main()
     } // end of smallprogram
